src/gtimelog/timelog.py

```diff
@@ -1118,15 +1121,34 @@ def load(self):
                     if ':' in line:  # tasks with group prefix
                         group, task = [s.strip() for s in line.split(':', 1)]
                         groups.setdefault(group, []).append(task)
+                        tasks.append(group + ': ' + task)
                     else:  # "other" tasks
-                        others.append(line.strip())
+                        others.append(line)
+                        tasks.append(line)
         except IOError:
             pass # the file's not there, so what?
         # append the "other" tasks at the end
         self.groups = list(groups.items())
         if others:
             self.groups.append((self.other_title, others))
+        self.tasklist = tasks
 
     def reload(self):
         """Reload the task list."""
         self.load()
+
+    def index(self, value):
+        """
+        Return the index of a value in the tasklist
+
+        Like a list's index() function but returns the length of the list if
+        the value isn't listed, so that those values are deemed at the end of
+        the list
```

Comment:
Okay, this is a bit of a trap for code reviewers then.  It looks like the list.index() or str.index(), but behaves differently.

I'd like it to be be renamed.  Maybe `order()`?

Reply:
Renamed and description adapted.